api/python/provisioner/commands/validator/config_sections.py

```diff
@@ -120,3 +122,36 @@ def _validate_node_count(self, num_of_nodes, parsed):  # noqa: C901
         except Exception as exc:
             raise ValueError(f"Config Failed to apply: {str(exc)}")
 
+    def _validate_config_devices(self,config,number_of_nodes) :
+        res= grains_get(
+            "hostname_status:Chassis",
+            targets=local_minion_id()
+            )[local_minion_id()]
+        if ( res['hostname_status:Chassis']=='server') :
+            devices = cmd_run(
+                "multipath -ll|grep mpath|sort -k2|cut -d' ' -f1|sed 's|mpath|/dev/disk/by-id/dm-name-mpath|g'|paste -s -d, -",
+                targets=ALL_MINIONS
+            )
+        else :
+            devices = cmd_run(
+                "lsblk -nd -o NAME -e 11|grep -v sda|sed 's|sd|/dev/sd|g'|paste -s -d, -",
+                targets=ALL_MINIONS
+            )
+        for section in config.sections() :
+
+            if ("srvnode" in section):
+                if (config.has_option(section,'storage.cvg.0.data_devices')):
+                    if 'default' in section:
+                        node_list = ['srvnode-'+str(num) for num in range(1, int(number_of_nodes)+1)]
```

Comment:
please add spaces between operator
`['srvnode-' + str(num) for num in range(1, int(number_of_nodes) + 1)]`